python/cudf/cudf/tests/test_parquet.py

```diff
@@ -2523,16 +2522,41 @@ def test_parquet_writer_decimal(decimal_type, data):
     assert_eq(gdf.to_pandas(nullable=True), got)
 
 
+# TODO: Make use of set_option context manager
+# once https://github.com/rapidsai/cudf/issues/12736
```

Comment:
We should prioritize https://github.com/rapidsai/cudf/issues/12736 as a requirement for pandas 2.0 support rather than take on technical debt here. It's not a hard feature to implement, from what I can tell. You already implemented it in this context manager, I think. 😄 Just might need some tweaks.

Reply:
Implemented https://github.com/rapidsai/cudf/pull/13511 to address it in `branch-23.08` itself since it is also a pandas-1.x feature.

Reply:
Updated the code to utilize `cudf.option_context`, hence resolving this comment.